packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1272,11 +1277,18 @@ async function handler(
       );
 
       if (reqBody.teamMemberEmail) {
-        const teamMember = eventTypeWithUsers.users.find((user) => user.email === reqBody.teamMemberEmail);
-        if (teamMember) {
-          luckyUsers.push(teamMember);
+        // If requested user is not a fixed host, assign the lucky user as the team member
+        if (!fixedUserPool.some((user) => user.email === reqBody.teamMemberEmail)) {
+          const teamMember = eventTypeWithUsers.users.find((user) => user.email === reqBody.teamMemberEmail);
+          if (teamMember) {
+            luckyUsers.push(teamMember);
+          }
```

Comment:
@CarinaWolli if the contact owner is a fixed host then let's still find a lucky user